<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
    <class>Form</class>
    <widget class="QWidget" name="Form">
        <property name="geometry">
            <rect>
                <x>0</x>
                <y>0</y>
                <width>800</width>
                <height>600</height>
            </rect>
        </property>
        <property name="windowTitle">
            <string>Form</string>
        </property>
        <layout class="QGridLayout" name="gridLayout_2">
            <item row="0" column="0">
                <layout class="QGridLayout" name="gridLayout">
                    <item row="0" column="2">
                        <layout class="QVBoxLayout" name="verticalLayout">
                            <item>
                                <widget class="QLabel" name="label">
                                    <property name="text">
                                        <string>Input file</string>
                                    </property>
                                </widget>
                            </item>
                            <item>
                                <widget class="QLineEdit" name="txtInput"/>
                            </item>
                            <item>
                                <widget class="QPushButton" name="btnBrowse">
                                    <property name="text">
                                        <string>Browse</string>
                                    </property>
                                </widget>
                            </item>
                            <item>
                                <widget class="Line" name="line">
                                    <property name="orientation">
                                        <enum>Qt::Horizontal</enum>
                                    </property>
                                </widget>
                            </item>
                            <item>
                                <widget class="QLabel" name="label_2">
                                    <property name="text">
                                        <string>Rotation</string>
                                    </property>
                                </widget>
                            </item>
                            <item>
                                <widget class="QComboBox" name="comboRotation">
                                    <item>
                                        <property name="text">
                                            <string>90° Clockwise</string>
                                        </property>
                                    </item>
                                    <item>
                                        <property name="text">
                                            <string>90° Counter clockwise</string>
                                        </property>
                                    </item>
                                    <item>
                                        <property name="text">
                                            <string>180° Flip vertically</string>
                                        </property>
                                    </item>
                                </widget>
                            </item>
                            <item>
                                <widget class="Line" name="line_4">
                                    <property name="orientation">
                                        <enum>Qt::Horizontal</enum>
                                    </property>
                                </widget>
                            </item>
                            <item>
                                <widget class="QPushButton" name="btnRotate">
                                    <property name="text">
                                        <string>Rotate PDF</string>
                                    </property>
                                </widget>
                            </item>
                            <item>
                                <spacer name="verticalSpacer_2">
                                    <property name="orientation">
                                        <enum>Qt::Vertical</enum>
                                    </property>
                                    <property name="sizeHint" stdset="0">
                                        <size>
                                            <width>20</width>
                                            <height>40</height>
                                        </size>
                                    </property>
                                </spacer>
                            </item>
                            <item>
                                <spacer name="horizontalSpacer_2">
                                    <property name="orientation">
                                        <enum>Qt::Horizontal</enum>
                                    </property>
                                    <property name="sizeHint" stdset="0">
                                        <size>
                                            <width>40</width>
                                            <height>20</height>
                                        </size>
                                    </property>
                                </spacer>
                            </item>
                        </layout>
                    </item>
                    <item row="0" column="1">
                        <widget class="Line" name="line_2">
                            <property name="orientation">
                                <enum>Qt::Vertical</enum>
                            </property>
                        </widget>
                    </item>
                    <item row="0" column="0">
                        <spacer name="horizontalSpacer">
                            <property name="orientation">
                                <enum>Qt::Horizontal</enum>
                            </property>
                            <property name="sizeHint" stdset="0">
                                <size>
                                    <width>40</width>
                                    <height>20</height>
                                </size>
                            </property>
                        </spacer>
                    </item>
                    <item row="0" column="4">
                        <spacer name="horizontalSpacer_3">
                            <property name="orientation">
                                <enum>Qt::Horizontal</enum>
                            </property>
                            <property name="sizeHint" stdset="0">
                                <size>
                                    <width>40</width>
                                    <height>20</height>
                                </size>
                            </property>
                        </spacer>
                    </item>
                    <item row="0" column="3">
                        <widget class="Line" name="line_3">
                            <property name="orientation">
                                <enum>Qt::Vertical</enum>
                            </property>
                        </widget>
                    </item>
                </layout>
            </item>
            <item row="1" column="0">
                <layout class="QHBoxLayout" name="horizontalLayout">
                    <item>
                        <spacer name="horizontalSpacer_4">
                            <property name="orientation">
                                <enum>Qt::Horizontal</enum>
                            </property>
                            <property name="sizeHint" stdset="0">
                                <size>
                                    <width>40</width>
                                    <height>20</height>
                                </size>
                            </property>
                        </spacer>
                    </item>
                    <item>
                        <widget class="QPushButton" name="btnBack">
                            <property name="text">
                                <string>Back</string>
                            </property>
                        </widget>
                    </item>
                </layout>
            </item>
        </layout>
    </widget>
    <resources/>
    <connections/>
</ui>
